### ISSUE TITLE
Can we make this work with ES2015 arrow functions?

### ISSUE THREAD

**Issue by lmammino** [scottcorgan/args-list#4]
as per title

**Comment by scottcorgan** [comment]
I don't think that will be too hard. Just messing with the regex. I don't a lot of time at this moment. Want to open a PR?

**Comment by lmammino** [comment]
Thanks for your quick reply @scottcorgan !
At the moment I sorted my problem out by using another module (I am in a rush :stuck_out_tongue_closed_eyes:), but if I can find some free time in the weekend I will be more than happy to contribute.

Regards
